render-utils.js

```diff
@@ -0,0 +1,15 @@
+export function renderWorkshop(workshop) {
+    const div = document.createElement('div');
+    const h2 = document.createElement('h2');
+    div.classList.add('workshop');
+    h2.textContent = workshop.name;
+    div.append(h2);
+    return div;
+}
+
+export function renderOption(workshop) {
+    const option = document.createElement('option');
+    option.value = workshop.id;
+    option.textContent = workshop.name;
+    return option;
+}
```

Comment:
I like how you kept these as a whole separate file to reduce your code in the workshop.js.